engine-spring/core/src/main/java/org/operaton/bpm/engine/spring/components/scope/ProcessScope.java

```diff
@@ -65,11 +64,9 @@ public class ProcessScope implements Scope, InitializingBean, BeanFactoryPostPro
 	public static final String PROCESS_SCOPE_PROCESS_VARIABLES_SINGLETON = "processVariables";
 	public static final String PROCESS_SCOPE_NAME = "process";
 
-	private ClassLoader classLoader = ClassUtils.getDefaultClassLoader();
-
-	private boolean proxyTargetClass = true;
```

Comment:
**Review Hint**: This private field was effective final with value `true`. There is no known use case for having it as a field.

Reply:
I think the use case was that the variable name carries semantics, otherwise you need to read the `Scopifier`s method signature. In general I'm a big fan of the approach as it makes the code easier to read